kraken-build/src/kraken/std/python/tasks/mypy_task.py

```diff
@@ -49,10 +46,10 @@ def get_execute_command_v2(self, env: MutableMapping[str, str]) -> list[str]:
         status_file = (self.project.directory / ".dmypy.json").absolute()
         if use_daemon:
             command += ["--status-file", str(status_file), "run", "--"]
-        if mypy_pex_bin:
+        if mypy_cmd:
             # Have mypy pick up the Python executable from the virtual environment that is activated automatically
             # during the execution of this task as this is an "EnvironmentAwareDispatchTask". If we don't supply this
-            # option, MyPy will only know the packages in its PEX.
+            # option, MyPy will only know the packages in its own virtual environment.
             command += ["--python-executable", "python"]
```

Comment:
This needs updating so mypy can access dependencies outside the `uv tool`-managed environment.